src/Tizen.Multimedia.MediaPlayer/Player/PlayerEnums.cs

```diff
@@ -132,6 +132,7 @@ public enum PlayerState
         /// <summary>
         /// Preparation in progress.
         /// </summary>
+        /// <remarks>Preparing state is not regular state. It's only for PrepareAsync.</remarks>
```

Comment:
I didn't use "regular" for "public". i meant it is not fixed clearly like other states. it just passes by.

Reply:
I am trying to decide whether or not i should put that sentence.
please tell me your view again.

/// <remarks>`Preparing state is not regular state.` It's only for PrepareAsync.
/// Other methods and properties cannot be set in this state.</remarks>

Reply:
I did not mean "regular" for "public" keyword itself also, but meant it's already regular state because it's introduced by `public` PlayerState.Preparing and it means user can access it with `State` property.

But it's true that it's translated to other state internally without any other state translating input. 
So If you want to document it, do it. But I still think it's better to remove the statement for `regular`.

How about documenting the `Preparing` state is translated to other state internally and, in this state, other methods and properties can not be set? 
The details are changed.

Reply:
It's different with other state but PlayerState.Preparing is already introduced so 'not regular' can make user confused.

How about documenting like below or simillar one? 
This state is set by PrepareAsync method and translated to other state internally. Other methods and properties can not be called or set in this state.